rust/private/clippy.bzl

```diff
@@ -180,10 +182,10 @@ rust_clippy = rule(
     attrs = {
         "deps": attr.label_list(aspects = [rust_clippy_aspect]),
     },
-    doc = """
+    doc = """\
```

Comment:
So there aren't unnecessary new lines when it gets rendered, I thought it would be a less invasive change then moving the first line here. Would you like me to do this instead?

Reply:
For the first line, it may looks nicer to remove it but keeping the file as is is good for me too.